lib/autoload/globals/Iter.ngs

```diff
@@ -355,3 +355,22 @@ TEST { {}.Iter().next() }.assert(NoNext)
 doc Convert Iter into JSON compatible data structure - array.
 doc Forces evaluation of MapIter for example.
 F JsonData(i:Iter) collector i.each(collect + JsonData)
+
+type SkipError(Error)
+
+doc Returns an Iter that provides all except the first count elements.
```

Comment:
Should be: Returns the given Iter after consuming first count elements

(Important to emphasize that it's the same one)